runtime/jvmMain/src/kotlinx/benchmark/JvmBenchmarkAnnotations.kt

```diff
@@ -20,11 +20,9 @@ actual typealias OutputTimeUnit = org.openjdk.jmh.annotations.OutputTimeUnit
 @Suppress("ACTUAL_WITHOUT_EXPECT")
 actual typealias BenchmarkTimeUnit = java.util.concurrent.TimeUnit
 
-@Suppress("ACTUAL_ANNOTATION_CONFLICTING_DEFAULT_ARGUMENT_VALUE", "NO_ACTUAL_CLASS_MEMBER_FOR_EXPECTED_CLASS")
 actual typealias Warmup = org.openjdk.jmh.annotations.Warmup
 
 @Suppress("ACTUAL_ANNOTATION_CONFLICTING_DEFAULT_ARGUMENT_VALUE")
 actual typealias Measurement = org.openjdk.jmh.annotations.Measurement
 
-@Suppress("NO_ACTUAL_CLASS_MEMBER_FOR_EXPECTED_CLASS")
-actual typealias Param = org.openjdk.jmh.annotations.Param
```

Comment:
The same here: https://github.com/Kotlin/kotlinx-benchmark/pull/154/files#r1350778685